### PR TITLE
[MAT-6835] DTOs for QRDA Export

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-6835](https://jira.cms.gov/browse/MAT-6835)
(Optional) Related Tickets:

### Summary

- **MAT-6835: Pass DTO with test case stats to QRDA service for top level summary file.**
- **MAT-6835: Reworking QRDA DTOs, removing pass/fail and moving coverage percentage into testCaseDTOs.**
- **MAT-6835: Use DTOs instead of Object[].**

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/dto/qrda/StratificationDTO.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class StratificationDTO {
```
Comment (chubert-sb) [inline src/main/java/gov/cms/madie/dto/qrda/StratificationDTO.java:12]: If these DTOs are going to be shared across multiple services we should probably eventually move them to madie-java-models or something. But that can be done later

**Review by chubert-sb — APPROVED** [review]

File: src/main/java/gov/cms/madie/dto/qrda/StratificationDTO.java (hunk 1 above)
Comment (jkotanchik-SB) [reply src/main/java/gov/cms/madie/dto/qrda/StratificationDTO.java:12]: Agreed. I should make a clean-up task to do that once the changes clear QA.
